tests/ui/macros/borrowck-error-in-macro.rs

```diff
@@ -0,0 +1,7 @@
+//@ aux-build: borrowck-error-in-macro.rs
+//@ error-pattern: a call in this macro requires a mutable binding due to mutable borrow of `d`
```

Comment:
It seems there is no way to match label text other than error-pattern :/
I might fix that separately later hehe

Reply:
Can you please leave a FIXME comment pointing to #141896?